### PR TITLE
Update /rest/db/ignores for 0.13 changes

### PR DESCRIPTION
**Pull request by imsodin**
I hope I got it right.

### CODE REVIEW

**Comment by AudriusButkevicius** [comment]
It's no longer patterns but expanded in the example dump, no?

File: rest/db-ignores-get.rst [hunk 1]
```diff
@@ -2,19 +2,17 @@ GET /rest/db/ignores
...
+provides a list of strings which represent globbing patterns described by gobwas/glob (based on standard wildcards) that match the patterns in ``.stignore`` and all the includes. If appropriate these globs are prepended by the following modifiers: ``!`` to negate the glob, ``(?i)`` to do case insensitive matching and ``(?d)`` to enable removing of ignored files in an otherwise empty directory.
 
 .. code-block:: json
 
     {
       "ignore": [
-        "/Backups"
+        "(?i)/Backups"
       ],
       "patterns": [
```
Comment (calmh) [inline rest/db-ignores-get.rst]: Here

**Comment by imsodin** [comment]
Corrected, I must have been temporarily blind - sorry...

**Comment by AudriusButkevicius** [comment]
@st-review merge

**Comment by st-review** [comment]
:ok_hand: Merged as c6b994a815ee85388d9231c5add31578bd182b27. Thanks, @imsodin!

**Pull request closed without merging**